templates/assembly-script/.github/workflows/release.hippo.yml

```diff
@@ -41,19 +41,23 @@ jobs:
     - name: Install Hippo CLI
       run: |
         mkdir -p tools/tmp
-        wget https://github.com/deislabs/hippo-cli/releases/download/v0.7.0/hippo-v0.7.0-linux-amd64.tar.gz -q -O ./tools/tmp/hippo.tar.gz
+        wget https://github.com/deislabs/hippo-cli/releases/download/v0.9.0/hippo-v0.9.0-linux-amd64.tar.gz -q -O ./tools/tmp/hippo.tar.gz
```

Comment:
Frowns upon not using https://github.com/marketplace/actions/engineerd-configurator for this.
(joking, of course, only use it if it simplifies the action).